runtime/tr.source/trj9/build/files/host/i386.mk

```diff
@@ -20,7 +20,6 @@
 
 JIT_PRODUCT_SOURCE_FILES+=\
     tr.source/trj9/x/i386/runtime/J9IA32ArrayCopy.asm \
-    tr.source/trj9/x/i386/runtime/J9IA32ArrayTranslate.asm \
     tr.source/trj9/x/i386/runtime/J9IA32CompressString.asm \
```

Comment:
Slightly related but it appears J9IA32CompressString.asm can be deprecated too.

Reply:
Thank-you for pointing it out, I'll look into clean it up later.